modules/bq-log-alerting/README.md

```diff
@@ -0,0 +1,146 @@
+# Log Export: BigQuery Log Alerting
+
+This submodule allows you to configure a BigQuery Log Alerting solution on GCP. It uses the log export created in the root module to create findings in the [Security Command Center](https://cloud.google.com/security-command-center) under a source called "BQ Log Alerts" based on rules defined as views in BigQuery.
+
+## Basic Architecture
+
+### Cloud Logging
+
+The high-level overview of the solution is that [log sinks](https://github.com/terraform-google-modules/terraform-google-log-export) send all Cloud Audit Logs and VPC Flow Logs to [BigQuery](https://github.com/terraform-google-modules/terraform-google-log-export/tree/master/modules/bigquery) located in a centralized logging project.
+Custom views in BigQuery are created that look for specific activities in these logs, defined by a SQL query, e.g. looking for events that match `v1.compute.routes.insert` or `v1.compute.routes.delete`.
+On a regular interval (`job_schedule` variable , default 15 minutes), Cloud Scheduler writes a message containing a time window parameter (`time_window_quantity` and `time_window_quantity` variables, default 20 minutes) to PubSub.
+This 15 minute schedule with 20 minute window is used to ensure some overlap between runs of the function, to catch cases where events may occur just as the Cloud Function run has kicked-off.
+The message in PubSub acts as the trigger for the Google Cloud Function (GCF) which reads from the views that exist (one for each use case) and writes any events it finds to Cloud Security Command Center (CSCC).
+These events are called "findings" in CSCC parlance and represent events that are actionable, e.g. can close them after investigation.
+Any duplicate findings are ignored, as the unique ID for the finding is generated from a series of constants describing a particular event, and is thus repeatable.
```

Comment:
Do we mention what these constants are?

Reply:
It is a MD5 hash generated from the concatenation of the bq view name + the eventTimestamp + the callerIp + the principalEmail + theresourceName.

Reply:
fixed